jre/guide/java/java_jre_accepted_sites_configured/java_jre_accepted_sites_exists/rule.yml

```diff
@@ -0,0 +1,37 @@
+documentation_complete: true
+
+title: 'The Java exception.sites File Exists'
+
+description: |-
+    If the Java accepted sites list file does not exist, it can be added
+    by running:
```

Comment:
```suggestion
    By default, no <tt>exception.sites</tt> file exists which means that there is no prevention of
    unauthorized software. The <tt>exception.sites</tt> file is a text file containing single-line URLs for
    accepted risk sites. If the Java accepted sites list file does not exist, it can be added
    by running:
```